 jemalloc: apply recommended optimizations

The Jemalloc project suggests using -O3 and -funroll-loops for optimization.
Let's enable these flags by default in order to improve the performance.
 enable -Os Aggressive -O2, -O3 optimizations 

diff --git a/Android.bp b/Android.bp
index f65c934..3d55e6f 100644
--- a/Android.bp
+++ b/Android.bp
@@ -17,7 +17,7 @@
 common_cflags = [
     "-std=gnu11",
     "-D_REENTRANT",
-    "-O3",
+    "-O2",
     "-funroll-loops",
     "-fvisibility=hidden",
     "-Wno-unused-parameter",
diff --git a/Android.mk b/Android.mk
index a8819f9..4062660 100644
--- a/Android.mk
+++ b/Android.mk
@@ -23,7 +23,7 @@ LOCAL_PATH := $(call my-dir)
 jemalloc_common_cflags := \
 	-std=gnu11 \
 	-D_REENTRANT \
-	-O3 \
+	-O2 \
 	-funroll-loops \
 	-fvisibility=hidden \
 	-Wno-unused-parameter \
diff --git a/Android.mk b/Android.mk
index a8819f9..4062661 100644
--- a/Android.mk
+++ b/Android.mk
@@ -16,10 +16,6 @@
 
 LOCAL_PATH := $(call my-dir)
 
-LOCAL_SDCLANG_LTO := true
-LOCAL_SDCLANG_LTO_LDFLAGS := -O3 -funroll-loops
-LOCAL_FDO_SUPPORT := true
-
 jemalloc_common_cflags := \
 	-std=gnu11 \
 	-D_REENTRANT \
